pkg/asset/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
These files shouldn't exist. Right? We don't need bazel for new installer binary.